projects/rero/ng-core/src/lib/record/editor/editor.component.ts

```diff
@@ -287,6 +274,34 @@ export class EditorComponent implements OnInit, OnDestroy {
     this.fields = fields;
   }
 
+  /**
+   * Hide of show the field depending on the model value.
+   * @param field formly field config
+   */
+  private hideShowEmptyField(field: FormlyFieldConfig) {
+    let model = field.model;
+    // for simple object the model is the parent dict
+    if (
+      !['object', 'multischema', 'array'].some(f => f === field.type)
+    ) {
+      model = field.model[field.key];
+    }
+    model = removeEmptyValues(model);
+    const modelEmpty = isEmpty(model);
+    if (!modelEmpty && (field.hide !== false)) {
+      setTimeout(() => {
+        field.hide = false;
+        this.editorService.removeHiddenField(field);
+      });
+    }
+    if (modelEmpty && (field.templateOptions.hide === true && field.hide === undefined)) {
```

Comment:
Why testing *field.hide === undefined* ? It's probably correct but I don't understand why..

Reply:
at the Initial step hide is undefined means untouched